kolibri/plugins/coach/assets/src/views/coach-exercise-detail-page/attempt-log-list.vue

```diff
@@ -1,44 +1,41 @@
 <template>
 
-  <div class="answer-history">
+  <div class="attempt-log-list">
     <h3 class="header">{{ $tr('header') }}</h3>
 
     <ul class="history-list">
-      <template v-for="(attemptLog, index) in attemptLogs">
-        <li v-if="index === 0">
-          <p class="item">
-            {{ daysElapsedText(attemptLog.daysElapsed) }}
-          </p>
-        </li>
+      <template v-for="attemptLog in attemptLogs">
+        <!--p v-if="index === 0" class="item">
+          TODO modify elapsed-time to do Days only?
+          {{ daysElapsedText(attemptLog.daysElapsed) }}
+        </p>
         <li v-else-if="attemptLogs[index - 1].daysElapsed != attemptLog.daysElapsed">
           <p class="item">
             {{ daysElapsedText(attemptLog.daysElapsed) }}
           </p>
-        </li>
-        <li @click="setSelected(index)" :class="isSelected(index)" class="clickable">
-          <div>
+        </li-->
```

Comment:
Yeup. Forgot to add the WIP tag. Serving as a reminder to use your `time-elapsed` component